pkg/services/ldap/settings_test.go

```diff
@@ -9,19 +9,43 @@ import (
 )
 
 func TestReadingLDAPSettings(t *testing.T) {
-	config, err := readConfig("testdata/ldap.toml")
+	actualConfig, err := readConfig("testdata/ldap.toml")
```

Comment:
Renamed `config` var because it was colliding with the global var from settings.go